/*
 * This file was automatically generated by EvoSuite
 * Sun Apr 16 16:21:23 GMT 2023
 */

package Model.Tests.Test;

import Model.Tests.Scaffolding.LogAlreadyExistsException_ESTest_scaffolding;
import org.junit.Test;
import Model.Exceptions.LogAlreadyExistsException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class LogAlreadyExistsException_ESTest extends LogAlreadyExistsException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      LogAlreadyExistsException logAlreadyExistsException0 = new LogAlreadyExistsException("");
  }
}
